provider/pkg/provider/remote/connection.go

```diff
@@ -46,7 +51,7 @@ type Connection struct {
 	PrivateKeyPassword *string  `pulumi:"privateKeyPassword,optional"`
 	AgentSocketPath    *string  `pulumi:"agentSocketPath,optional"`
 	DialErrorLimit     *int     `pulumi:"dialErrorLimit,optional"`
-	PerDialTimeout     int      `pulumi:"perDialTimeout,optional"`
+	PerDialTimeout     *int     `pulumi:"perDialTimeout,optional"`
```

Comment:
This is necessary?

Reply:
It's not necessary, but it is more consistent with the style of the provider and the other optional values near it.